podpac/core/interpolation/test/test_interpolation.py

```diff
@@ -23,13 +21,15 @@
 from podpac.core.compositor.tile_compositor import TileCompositorRaw
 from podpac.core.interpolation.scipy_interpolator import ScipyGrid
 
+_CRS = "+proj=longlat +datum=WGS84 +no_defs +vunits=m"
+
 
 class TestInterpolationMixin(object):
     def test_interpolation_mixin(self):
         class InterpArray(InterpolationMixin, ArrayRaw):
             pass
 
-        data = np.random.rand(4, 5)
+        data = np.ones(shape=(4, 5))
```

Comment:
Please either fix the random seed, OR use 'arange' to make sure the data varies. 

Repeat this comment for the rest of the file.